backend/package.json

```diff
@@ -3,16 +3,14 @@
   "version": "1.7.5",
   "description": "youtube.tracking.exposed - backend data collector",
   "scripts": {
-    "watch": "key=fuffa DEBUG=*,-express:*,-body-parser:*,-follow-redirects node_modules/.bin/ts-node-dev --respawn --transpile-only bin/server.js",
-    "start": "gitlog=`git log -30 --stat --oneline --decorate --graph` DEBUG=*,-body-parser:*,-express:*,-lib:*,-follow-redirects ts-node bin/server",
+    "watch": "key=fuffa DEBUG=*,-express:*,-body-parser:*,-follow-redirects node_modules/.bin/ts-node-dev -r tsconfig-paths/register --respawn --transpile-only bin/server.js",
+    "start": "gitlog=`git log -30 --stat --oneline --decorate --graph` DEBUG=*,-body-parser:*,-express:*,-lib:*,-follow-redirects ts-node -r tsconfig-paths/register bin/server",
     "parserv": "DEBUG=*,-pubtimeAPI node_modules/.bin/ts-node bin/parserv2.js",
     "parserv:watch": "DEBUG=*,-pubtimeAPI node_modules/.bin/ts-node-dev --respawn --transpile-only bin/parserv2.js",
     "leaveserv": "DEBUG=* node bin/leaveserv.js",
     "leaveserv:watch": "DEBUG=* node_modules/.bin/ts-node-dev --respawn --transpile-only bin/leaveserv.js",
     "test": "mocha --require ts-node/register tests/basic/*",
-    "campaigns": "node bin/campaignUpdater.js --dest https://youtube.tracking.exposed",
-    "build": "tsc --build",
-    "build:watch": "tsc --build -w"
```

Comment:
I would leave one script to check the code compiles without starting the server, maybe:
`"type-check": "tsc
`

Reply:
done